CHANGELOG.md

```diff
@@ -1,7 +1,13 @@
 ### Main (unreleased)
 
-- Added CSV batching functionality to EnumeratorBuilder with `build_csv_enumerator_on_batches` method and `csv_on_batches` alias.
-- Added support for custom timezones on ActiveRecordEnumerator and ActiveRecordBatchEnumerator. This allows for using cursors with datetime columns where ActiveRecord.default_timezone is set to :local and ActiveRecord is not using the same timezone as the database.
+Nil
+
+## v1.7.0 (Oct 11, 2024)
+
+### Features
+
+- [509](https://github.com/Shopify/job-iteration/pull/509) - Added CSV batching functionality to EnumeratorBuilder with `build_csv_enumerator_on_batches` method and `csv_on_batches` alias.
+- [512](https://github.com/Shopify/job-iteration/pull/512) - Added support for custom timezones on ActiveRecordEnumerator and ActiveRecordBatchEnumerator. This allows for using cursors with datetime columns where ActiveRecord.default_timezone is set to :local and ActiveRecord is not using the same timezone as the database.
```

Comment:
Curious if you think anything here should be formatted in backticks for readability